x-pack/plugins/canvas/canvas_plugin_src/functions/common/compare.js

```diff
@@ -4,10 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 export const compare = () => ({
   name: 'compare',
-  help:
-    'Compare the input to something else to determine true or false. Usually used in combination with `{if}`. This only works with primitive types, such as number, string, and boolean.',
+  help: i18n.translate('xpack.canvas.functions.compareHelpText', {
+    defaultMessage:
+      'Compare the input to something else to determine true or false. Usually used in combination with `{if}`. This only works with primitive types, such as number, string, and boolean.',
```

Comment:
put {if} to values. defaultMessage should not contain braces if it does not expect some param from values